commit 34604d28916710070390036118fcd21217b0f597
Author: Tony Luck <tony.luck@intel.com>
Date:   Fri May 6 15:54:10 2022 -0700

    Documentation: In-Field Scan
    
    Add documentation for In-Field Scan (IFS). This documentation
    describes the basics of IFS, the loading IFS image, chunk
    authentication, running scan and how to check result via sysfs.
    
    The CORE_CAPABILITIES MSR enumerates whether IFS is supported.
    
    The full  github location for distributing the IFS images is
    still being decided. So just a placeholder included for now
    in the documentation.
    
    Future CPUs will support more than one type of test. Plan for
    that now by using a "_0" suffix on the ABI directory names.
    Additional test types will use "_1", etc.
    
    Reviewed-by: Dan Williams <dan.j.williams@intel.com>
    Signed-off-by: Tony Luck <tony.luck@intel.com>
    Reviewed-by: Thomas Gleixner <tglx@linutronix.de>
    Link: https://lore.kernel.org/r/20220506225410.1652287-13-tony.luck@intel.com
    Signed-off-by: Hans de Goede <hdegoede@redhat.com>

diff --git a/Documentation/x86/index.rst b/Documentation/x86/index.rst
index 91b2fa456618..9d8e8a73d57b 100644
--- a/Documentation/x86/index.rst
+++ b/Documentation/x86/index.rst
@@ -35,6 +35,7 @@ x86-specific Documentation
    usb-legacy-support
    i386/index
    x86_64/index
+   ifs
    sva
    sgx
    features